src/libostree/ostree-repo-refs.c

```diff
@@ -1295,7 +1326,7 @@ ostree_repo_list_collection_refs (OstreeRepo                 *self,
 
   g_string_truncate (base_path, 0);
 
-  for (const char **iter = refs_dirs; iter && *iter; iter++)
+  for (const char **iter = (const char **)refs_dirs->pdata; iter && *iter; iter++)
```

Comment:
Since we have a `GPtrArray` now, it'd be cleaner to not append a final `NULL` and use `refs_dir->len`. But eh, not worth a respin!